### PR TITLE
Add text file for swamp nouns

### PR DESCRIPTION
**Pull request by dmtrek14**
Just to note: I didn't make changes to the forest.txt files. I merged those in from the upstream repo's main branch.

### CODE REVIEW

File: name-segments/nouns/swamp.txt [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+salamander
+crab
+conch
+fish
+snake
+heron
+crane
+loon
+fly
+dragonfly
```
Comment (ianfhunter) [inline name-segments/nouns/swamp.txt:32]:
```suggestion
mud
muck
```

File: name-segments/nouns/swamp.txt [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+river
+plain
+tree
+bog
+cypress
+reed
+grass
+sedge
+willow
+mangrove
```
Comment (ianfhunter) [inline name-segments/nouns/swamp.txt:14]:
```suggestion
grove
```

**Review by ianfhunter — APPROVED** [review]
LGTM, Thanks!
Just edited some that were a bit long

**Comment by dmtrek14** [comment]
@ianfhunter would you mind tagging this one as hacktoberfest-accepted so it might count for the challenge?

**Pull request merged**